networkx/drawing/layout.py

```diff
@@ -663,8 +671,10 @@ def kamada_kawai_layout(G, dist=None,
                 continue
             dist_mtx[row][col] = rdist[nc]
 
-    if pos is None:
+    if pos is None and dim >= 2:
         pos = circular_layout(G, dim=dim)
+    elif pos is None:
+        pos = {n: pt for n,pt in zip(G, np.linspace(0, 1, len(G)))}
```

Comment:
Check for PEP8 code style (space after comma and other things) perhaps using the pep8 command line tool. install with ```pip install pep8```, or read the Python PEP8 document itself.

Also, you can avoid evaluating ```pos is None``` twice by nesting your if statements. Not a big deal either way.